pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -172,15 +172,29 @@ func NewListOptionIndexer(ctx context.Context, s Store, opts ListOptionIndexerOp
 		return nil, err
 	}
 
+	fieldSet := make(map[string]struct{})
```

Comment:
We want to avoid duplicated fields here.

We don't support multiple columns pointing to the same json path. Adding support will require _more_ work from both UI and backend -> We need to have an ID for a field and its json path (or get rid of the json path altogether, but that brings its own set of complications).